kernel/kernel-util/src/main/java/org/sakaiproject/util/BaseResourceProperties.java

```diff
@@ -137,7 +137,7 @@ public BaseResourceProperties(Element el)
 					// if we don't have a value yet, make a list to hold this one
 					if (current == null)
 					{
-						List values = new Vector();
+						List<String> values = new Vector();
```

Comment:
This should be new Vector<>() to turn on inference

Reply:
OK

Reply:
`Vector` is deprecated, can't we just make this an `ArrayList`?